### DISCUSSION TITLE
Why mill generates mill-debug-log.txt file at my home directory

### DISCUSSION THREAD

**Discussion by dragove** [com-lihaoyi/mill#4010] [Q&A] [answered]
I found that mill generates mill-debug-log.txt file at my home directory. From the documentation, if I understand correctly, only `mill.main.client.DebugLog.println` can produce that file. Maybe I'm doing something wrong here.

Here are my environments:

My OS: Arch Linux

I'm using the mill bootstrap script by following command

```shell
curl -L https://github.com/com-lihaoyi/mill/releases/download/0.12.2/0.12.2 > mill && chmod +x mill
```

and here is my `build.sc` file

```scala
import mill._, scalalib._

trait CommonModule extends ScalaModule {
  def scalaVersion = "3.6.1"
  def scalacOptions = Seq(
    "-Werror",
    "-feature",
    "-Wsafe-init",
    "-Wunused:all",
    "-deprecation",
    "-experimental",
    "-Yexplicit-nulls",
    "-language:strictEquality",
  )
}

object bifer extends CommonModule {
  val sqalaVersion = "0.0.47"
  def ivyDeps = Agg(
    ivy"com.wz7982::sqala-dsl:$sqalaVersion",
    ivy"com.wz7982::sqala-jdbc:$sqalaVersion",
    ivy"com.wz7982::sqala-dynamic:$sqalaVersion",
    ivy"com.softwaremill.sttp.tapir::tapir-core:1.11.2",
    ivy"com.zaxxer:HikariCP:5.1.0",
    ivy"org.postgresql:postgresql:42.7.4",
    ivy"org.apache.logging.log4j:log4j-slf4j2-impl:2.23.1"
  )

  object test extends ScalaTests with TestModule.Munit {
    def ivyDeps = Agg(
      ivy"org.scalameta::munit:1.0.1"
    )
  }
}
```

**Comment by lihaoyi** [comment, marked as answer]
Looks like an leftover debug statement during development, will get rid of it in the next release https://github.com/com-lihaoyi/mill/commit/a458d471abd9b9a00135e8c5fa134b8870f226a2#diff-47934c0b243e400ed852f6763be9b5309d1319188aff3218a2813d4b2576f81bR42

**Reply by dragove** [reply]
Thanks For your quick reply. I tested it with version `0.12.2-40-b240f5` and there is no such file generated. I finally found it was already fixed in https://github.com/com-lihaoyi/mill/commit/7457601efdc5c959aef8bfd7bcb8aeb2513cdab3. Anyway, next release can solve this. Thanks.
